### DISCUSSION TITLE
Conditional card should allow `condition`

### DISCUSSION THREAD

**Discussion by sciurius** [home-assistant/frontend#13267] [Dashboard]
Home Assistant supports https://www.home-assistant.io/docs/scripts/conditions. However, the conditional card uses its own `conditions` that are limited to `and`ing state conditons.

**Comment by gonzotek** [comment]
Big +1 on this.  Just today I _really_ wanted to use some attributes for conditions in a dashboard. It would be lovely if condition cards and condition rows in entities cards could use as many of the same script/automation conditions and their options as makes sense (looking at the docs, I think only trigger mightn't be applicable to a dashboard context).
